### DISCUSSION TITLE
Dose in NCA output?

### DISCUSSION THREAD

**Discussion by kylebaron** [humanpred/pknca#334] [Q&A] [answered]
Asking this in Discussion since I'm not sure if it is already available or not: can you include the dose used in `PKNCA::pk.calc.cl()` or other calculations into the output? I've got some "odd" dosing in my data set and I wondered what dose was getting used to calculate clearance. Thank you in advance for pointing me in the right direction if this is already provided somewhere!

Kyle

**Comment by billdenney** [comment, marked as answer]
Yes, you can use the `totdose` NCA parameter to get the dose administered. A reference of all available parameters is here: https://billdenney.github.io/pknca/articles/v03-selection-of-calculation-intervals.html#parameters-available-for-calculation-in-an-interval

(I understand that the parameter list is long. We have some ideas to simplify parameter selection for future versions. Input is welcome.)

**Reply by kylebaron** [reply]
Thanks, @billdenney. I see it now. I had found the list, but just didn't search well-enough. Does just what I need.

**Reply by billdenney** [reply]
I'm glad that it helped, @kylebaron. If you have any suggestions to make it easier to find what you're looking for, I'd love to hear it. I have an active effort to make PKNCA easier to use now, and getting feedback on what is easy/hard would help.

**Reply by kylebaron** [reply]
I don't have a better idea than what you've already done. The information was there to find; I don't know what else you could do as the developer. I sometimes see people getting frustrated with too much documentation, but what can you do? If you remove it then there's _no_ documentation. It's a fine line and sometimes people just need to ask for help and get pointed in the right place.

**Reply by billdenney** [reply]
Thanks for that feedback. If you see any other areas for improvement, please let me know.
